pkg/cri/opts/spec_linux.go

```diff
@@ -139,19 +140,26 @@ func IsCgroup2UnifiedMode() bool {
 }
 
 // WithCDI updates OCI spec with CDI content
-func WithCDI(annotations map[string]string) oci.SpecOpts {
+func WithCDI(annotations map[string]string, CDIDevices []*runtime.CDIDevice) oci.SpecOpts {
 	return func(ctx context.Context, _ oci.Client, c *containers.Container, s *oci.Spec) error {
-		// TODO: Once CRI is extended with native CDI support this will need to be updated...
-		_, cdiDevices, err := cdi.ParseAnnotations(annotations)
-		if err != nil {
-			return fmt.Errorf("failed to parse CDI device annotations: %w", err)
-		}
-		if cdiDevices == nil {
-			return nil
+		var cdiDevices []string
```

Comment:
nit ...naming is hard :-) I think using the same name with a different case might cause a mistake some day...

Reply:
@mikebrow Thank you for the review! I agree, the naming was confusing. Updated.